lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1602,6 +1590,22 @@ def cmd_set(*args)
     name  = args[0]
     value = args[1, args.length-1].join(' ')
 
+    set_option(name, value, global: global, append: append)
+  end
+
+  #
+  # Sets a name to a value in a context aware environment.
+  #
+  def set_option(name, value, global: false, append: false)
```

Comment:
Can we revert this file now? 🤔

Reply:
@dwelch-r7 Just confirming if this file can be reverted or not :+1:

Reply:
It's not needed for these changes no, but I think they're good changes, I can revert and put it up as a separate PR, that probably makes more sense